core/src/main/resources/jenkins/model/Jenkins/login.jelly

```diff
@@ -50,6 +50,7 @@ THE SOFTWARE.
   <!-- in case of error we want to surround the form elements with an error hint -->
   <j:set var="inputClass" value="${error ? 'danger' : 'normal'}"/>
   <j:set var="simpleDecorator" value="${h.simplePageDecorator}"/>
```

Comment:
It looks to me this should be deleted, given AFAIU now `simpleDecorator` is not used anymore ?